docs/v3/get-started/quickstart.mdx

```diff
@@ -1,179 +1,125 @@
+
+<Steps>
+  <Step title="Create and login to your Prefect Cloud account.">
+    For the ease of getting started, we'll use a hosted Prefect server on Prefect Cloud. To create an account, we need to install `prefect-cloud`. We'll leverage `uv`, a modern Python package manager, to do this. Run the following commands in your terminal, after which you'll be prompted in your browser to create or login to your free Prefect Cloud account.
+
+    ```bash
+    curl -LsSf https://astral.sh/uv/install.sh | sh # Install `uv`.
+    uvx prefect-cloud login # Installs `prefect-cloud` into a temporary virtual env.
+    ```
+  </Step>
+  <Step title="What code should run?">
+
+    Let’s start with a boring Python script that fetches a list of customer ids and processes them all in parallel. You can find this code and other examples hosted in our [quickstart repository](https://github.com/PrefectHQ/quickstart/blob/main/01_getting_started.py). 
+
+   ```python 01_getting_started.py [expandable]
+    from prefect import flow, task
+    import random
+
+    @task
+    def get_customer_ids() -> list[str]:
+        # Fetch customer IDs from a database or API
+        return [f"customer{n}" for n in random.choices(range(100), k=50)]
+
+    @task
+    def process_customer(customer_id: str) -> str:
+        # Process a single customer
+        return f"Processed {customer_id}"
+
+    @flow
+    def main() -> list[str]:
+        customer_ids = get_customer_ids()
+        # Map the process_customer task across all customer IDs
+        results = process_customer.map(customer_ids)
+        return results
+
+
+    if __name__ == "__main__":
+        main()
+
+    ```
+
+    This looks like the ordinary Python you'd write, except there is 
+        - a `@flow` decorator on the script’s entrypoint.
+        - a `@task` decorator on each function called within the flow.
+
+    When you run this `flow`, Prefect dynamically creates a graph of each `task` and the state of their upstream dependencies. This allows Prefect to execute each `task` in the right order and, in the case of failure, to recover the state of your workflow and resume from its point of failure. 
+    
+    Let's run this code locally on your computer. To do that, run the following command in your terminal.
+    ```bash
+    git clone https://github.com/PrefectHQ/quickstart && cd quickstart
+    uv run 01_getting_started.py
+    ```
+    If all went well, you'll see a link to see its execution graph in the Prefect UI followed by a flurry of state changes and logs as Prefect dynamically discovers and executes your workflow.
+
+    ```bash {11} [expandable]
+    00:30:53.633 | INFO    | Flow run 'airborne-ringtail' - Beginning flow run 'airborne-ringtail' for flow 'main'
+    00:30:53.638 | INFO    | Flow run 'airborne-ringtail' - View at https://app.prefect.cloud/account/...
```

Comment:
I almost want there to be maybe less tasks? Only so the link to go see the flow run is much more clearly visible.

![Screenshot 2025-04-04 at 11 29 36 AM](https://github.com/user-attachments/assets/7440bbaf-83cb-443a-b23d-f330b9dc6859)